src/network/networkd-json.c

```diff
@@ -1044,12 +1074,102 @@ static int dhcp6_client_append_json(Link *link, JsonVariant **v) {
                 return 0;
 
         r = dhcp6_client_lease_append_json(link, &w);
+        if (r < 0)
+                return r;
+        r = dhcp6_client_pd_append_json(link, &w);
         if (r < 0)
                 return r;
 
         return json_append_one(v, "DHCPv6Client", w);
 }
 
+static int dhcp_client_lease_append_json(Link *link, JsonVariant **v) {
+        _cleanup_(json_variant_unrefp) JsonVariant *w = NULL;
+        uint32_t t1, t2;
+        int r;
+
+        assert(link);
+        assert(v);
+
+        if (!link->dhcp_lease)
+                return 0;
+
+        r = sd_dhcp_lease_get_t1(link->dhcp_lease, &t1);
+        if (r < 0)
+                return r;
+
+        r = sd_dhcp_lease_get_t2(link->dhcp_lease, &t2);
+        if (r < 0)
+                return r;
+
+        r = json_build(&w, JSON_BUILD_OBJECT(
+                                JSON_BUILD_PAIR_FINITE_USEC("T1", sec_to_usec(t1, 0)),
+                                JSON_BUILD_PAIR_FINITE_USEC("T2", sec_to_usec(t2, 0))));
+        if (r < 0)
+                return r;
+
+        return json_append_one(v, "Lease", w);
+}
+
+static int dhcp_client_pd_append_json(Link *link, JsonVariant **v) {
+        _cleanup_(json_variant_unrefp) JsonVariant *addresses = NULL, *array = NULL;
+        uint8_t ipv4masklen;
+        uint8_t sixrd_prefixlen;
```

Comment:
Please merge the declaration into one line.

Reply:
Done.